libs/eo/landing-page/shell/src/lib/eo-landing-page.integration.spec.ts

```diff
@@ -27,7 +27,14 @@ describe('EnergyOrigin landing page', () => {
   const findEnergyOriginLogo = () =>
     screen.findByRole('img', { name: 'EnergyOrigin' });
 
-  it('displays the EnergyOrigin logo', async () => {
+  const findEnergiNetLogo = () =>
+    screen.findByRole('img', { name: 'Energinet' });
+
+  const findFooterTelLink = () => screen.findByRole('link', { name: 'phone' });
```

Comment:
suggestion: Rename (update references as well) to query name which doesn't mention where the element has to be located. Use case-insensitive accessible name selector.
```suggestion
  const findPhoneLink = () => screen.findByRole('link', { name: /phone/i });
```